### PR TITLE
6.4 About page

### PR DESCRIPTION
**Pull request by luminuu**
Trac ticket: https://core.trac.wordpress.org/ticket/59289

This PR will need a few iterations. It's updating the About page for 6.4.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -55,56 +55,44 @@
...
+					<?php _e( 'Every version of WordPress is crafted to empower your creative freedom, and WordPress 6.4 is no different. New features and updates to your site editing, design, and writing experience allow your ideas to take shape seamlessly. Elevate your site-building journey with the flexibility and power of WordPress 6.4.' ); ?>
 				</p>
 			</div>
 		</div>
 
-		<div class="about__section has-2-columns">
-			<div class="column is-vertically-aligned-center">
+		<div class="about__section has-2-columns has-accent-4-background-color">
+			<div class="column is-vertically-aligned-center" style="padding-right: 6rem">
+				<h3><?php _e('Say hello to<br>Twenty Twenty-Four'); ?></h3>
```
Comment (mukeshpanchal27) [inline src/wp-admin/about.php]:
```suggestion
				<h3><?php _e( 'Say hello to<br>Twenty Twenty-Four' ); ?></h3>
```

PHPCS fix.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -55,56 +55,44 @@
...
+				<p><?php _e('Experience the latest advancements in site editing with Twenty Twenty-Four. Built with three distinct use cases in mind, the versatility of the new default theme makes it an ideal choice for almost any type of website. Dive into its collection of templates and patterns and unlock a world of creative possibilities with just a few tweaks.'); ?></p>
```
Comment (mukeshpanchal27) [inline src/wp-admin/about.php]:
```suggestion
				<p><?php _e( 'Experience the latest advancements in site editing with Twenty Twenty-Four. Built with three distinct use cases in mind, the versatility of the new default theme makes it an ideal choice for almost any type of website. Dive into its collection of templates and patterns and unlock a world of creative possibilities with just a few tweaks.' ); ?></p>
```

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks @luminuu for the PR. Left feedback that fix PHPCS fix.

File: src/wp-admin/about.php (hunk 1 above)
Comment (luminuu) [reply src/wp-admin/about.php]: Done!

File: src/wp-admin/about.php (hunk 2 above)
Comment (luminuu) [reply src/wp-admin/about.php]: Done!

**Comment by aaronjorbin** [comment]
I'm really impressed with my ability to screw up the git history here.  Going to do a new PR.

**Comment by aaronjorbin** [comment]
This was committed in https://core.trac.wordpress.org/changeset/56950

**Pull request closed without merging**